tests/Database/PgSqlDriver.formatLike.phpt

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * @dataProvider? databases.ini  postgresql
+ */
+
+use Tester\Assert,
+	Nette\Utils\DateTime;
+
+require __DIR__ . '/connect.inc.php'; // create $connection
+
+
+$tests = function($connection) {
+	$driver = $connection->getSupplementalDriver();
+	Assert::false($connection->query("SELECT 'AAxBB' LIKE", $connection::literal($driver->formatLike('A_B', 0)))->fetchField());
+	Assert::true( $connection->query("SELECT 'AA_BB' LIKE", $connection::literal($driver->formatLike('A_B', 0)))->fetchField());
+
+	Assert::false($connection->query("SELECT 'AAxBB' LIKE", $connection::literal($driver->formatLike('A%B', 0)))->fetchField());
+	Assert::true( $connection->query("SELECT 'AA%BB' LIKE", $connection::literal($driver->formatLike('A%B', 0)))->fetchField());
+
+	Assert::same('AA\\BB', $connection->query("SELECT 'AA\\BB'")->fetchField());
+	Assert::false($connection->query("SELECT 'AAxBB' LIKE", $connection::literal($driver->formatLike('A\\B', 0)))->fetchField());
+	Assert::true( $connection->query("SELECT 'AA\\BB' LIKE", $connection::literal($driver->formatLike('A\\B', 0)))->fetchField());
+};
```

Comment:
What about this tests?
```php
	$driver = $connection->getSupplementalDriver();
	Assert::false($connection->query("SELECT 'AAxBB' LIKE", $connection::literal($driver->formatLike('A_B', 0)))->fetchField());
	Assert::true( $connection->query("SELECT 'AA_BB' LIKE", $connection::literal($driver->formatLike('A_B', 0)))->fetchField());

	Assert::false($connection->query("SELECT 'AAxBB' LIKE", $connection::literal($driver->formatLike('A%B', 0)))->fetchField());
	Assert::true( $connection->query("SELECT 'AA%BB' LIKE", $connection::literal($driver->formatLike('A%B', 0)))->fetchField());

	Assert::false($connection->query("SELECT ? LIKE", 'AAxBB', $connection::literal($driver->formatLike('A\\B', 0)))->fetchField());
	Assert::true( $connection->query("SELECT ? LIKE", 'AA\\BB', $connection::literal($driver->formatLike('A\\B', 0)))->fetchField());

	Assert::false($connection->query("SELECT ? LIKE", 'AAxBB', $connection::literal($driver->formatLike("A'B", 0)))->fetchField());
	Assert::true( $connection->query("SELECT ? LIKE", "AA'BB", $connection::literal($driver->formatLike("A'B", 0)))->fetchField());

	Assert::false($connection->query("SELECT ? LIKE", 'AAxBB', $connection::literal($driver->formatLike('A"B', 0)))->fetchField());
	Assert::true( $connection->query("SELECT ? LIKE", 'AA"BB', $connection::literal($driver->formatLike('A"B', 0)))->fetchField());
```

The quotes escaping test is covered by [another test](https://github.com/dg/dibi/blob/020b15c0e29fd14d85dd510d560795d6d409cb6b/tests/dibi/DibiTranslator.phpt#L205) in dibi.